src/server/cli/test-runner.js

```diff
@@ -96,6 +96,13 @@ async function getTestRunnerOptions(argv = []) {
 			longAlias: 'webkit',
 			type: Boolean
 		},
+		{
+			name: 'slow',
+			alias: 's',
+			type: Number,
+			description: 'Tests whose duration in milliseconds are at most half of this threshold are "fast" and tests which exceed it are "slow"\n[Default: 75]',
```

Comment:
Maybe worth calling out both defaults (500 for vdiff, 75 for everyone else)?

Reply:
Cool -- done.